contributing/contributing-code.md

```diff
@@ -175,3 +175,36 @@ These guidelines are specific to the amphtml repository. Other ampproject repos
+
+**The bar for getting a cherry-pick into a live release is very high** because our goal is to produce high quality launches on a predictable schedule.
+
+**Keep in mind that performing a cherry-pick requires a significant amount of work from you and the on-duty engineer** and they can take a long time to process.
+
+- In general only fixes for [P0 issues](https://github.com/ampproject/amphtml/blob/master/contributing/issue-priorities.md) (causing "an outage or a critical production issue") may be cherry-picked. P0 issues are those that:
+  - cause privacy or security issues
+  - cause user data loss
+  - break existing AMP web pages in a significant way
+  - or would otherwise cause a significant harm to AMP's reputation if left unresolved
+- Regressions found in the **experimental**/**beta** releases that are not P0 _may_ be approved if they can be resolved with a rollback. Fixes other than rollbacks--no matter how simple they may seem--will not be approved because these have the potential to cause cascading problems and delay the release promotion of **beta** to **stable** for everyone.
+
+### Process for requesting a cherry-pick
+
+Use the following process to request a cherry-pick if you have a change that you believe meets the cherry-pick criteria.
+
+If you run into any issues or have any questions when requesting a cherry-pick, please use the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+
+- Ensure there is a GitHub issue filed for the problem that needs to be resolved _before_ filing the cherry-pick request issue.
+- File the cherry-pick request issue using the [Cherry-pick request template](https://github.com/ampproject/amphtml/issues/new?title=%F0%9F%8C%B8%20Cherry%20pick%20request%20for%20%3CYOUR_ISSUE_NUMBER%3E%20into%20%3CRELEASE_ISSUE_NUMBER%3E%20%28Pending%29&template=cherry_pick_template.md). Follow the instructions in the template, providing all of the requested data. **Make sure you fill out this issue completely or your cherry-pick may not be seen or acted upon.**
+- Get the necessary approval for your cherry-pick (indicated via comments on the cherry-pick request issue).
+  - For cherry-picks into canary, at least one member of the [Approvers WG](https://github.com/orgs/ampproject/teams/wg-approvers/members) must approve the cherry-pick.
+  - For cherry-picks into prod:
+    - if the fix is a clean rollback that does not require any other changes, at least one member of the [Approvers WG](https://github.com/orgs/ampproject/teams/wg-approvers/members) must approve the cherry-pick.
+    - otherwise, at least one member from the [Cherry-Pick Approvers group](https://github.com/orgs/ampproject/teams/cherry-pick-approvers/members) must approve the cherry-pick.
+- After the cherry-pick has been approved, the person currently handling releases (on-duty engineer) will work with you to ensure the cherry-pick is made.
```

Comment:
suggestion:
"Once approved, the on-duty engineer handling releases will work with you to ensure the cherry-pick is made. "

Reply:
Done (in follow-up)